src/services/legal-services.ts

```diff
@@ -126,7 +127,8 @@ export default class LegalServices {
       url += '?draft=true'
     }
 
-    return axios.put(url, { filing }).then(response => {
+    return axios.put(url, { filing },
+      { headers: { 'accountId': BusinessLookupServices.accountId } }).then(response => {
```

Comment:
Should the header property be named `Account-Id` ?

Reply:
we could. only thing is the existing api code expects accountId

Reply:
OK. I was hoping the APIs would be consistent. Resolving.